app/src/main/java/com/endava/myendava/rest/RetrofitServiceApi.java

```diff
@@ -19,6 +20,9 @@
 
 public interface RetrofitServiceApi {
 
+    @GET("user/login/{email}")
+    Observable<LoginResult> userLoginMock(@Path("email")String email);
```

Comment:
use a more specific name for this method. Something like checkIfUserIsValid or something ^^